fairlearn/reductions/_grid_search/_grid_generation.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
should we name this file `_grid_generator.py` similarly to `_threshold_optimizer.py` we do elsewhere?